### ISSUE TITLE
Manifest Ghost balance

### ISSUE THREAD

**Issue by Durgen** [d3athrow/vgstation13#14126]
#### Description of issue

Manifested ghosts need some balance, they are prone to fuckery
suggested balances
\>5 minute cooldown to becoming ghost, (so the ghosts aren't just an endless rape train)
\>Diona can't summon ghosts (so the ghosts aren't just an endless rape train)

- [X] a healthy portion of salt is motivating this issue

**Comment by Kurfursten** [comment]
>Races that don't suffer from blood sacrifice
>Able to use blood magic

It's not a sacrifice if there's no cost

**Comment by AutismSpider** [comment]
make it so races with the no blood flag can't make runes unless they use bottle of blood etc or there blood on the floor for them to use.

**Comment by Durgen** [comment]
![ooc](https://cloud.githubusercontent.com/assets/19157295/23140397/fac92290-f776-11e6-9e76-bff1398bb895.png)

**Comment by ghost** [comment]
just make it a 30 second cooldown and make the DPS burn damage
diona would still be able to manifest ghost but they'll take more damage

**Comment by AutismSpider** [comment]
@Durgen only two ways to fix this take out manifest ghost or make it do real fucking damage like clone/ massive burn and brute

**Comment by Durgen** [comment]
someone suggested in deadchat that there could be a 5 minute cool down for players becoming ghosts again
I think that would go a long way

**Comment by SonixApache** [comment]
![fuckyou](https://puu.sh/uc5AW/474d8b57ed.png)

Step 1: manifest ghost rune kills dionae
step 2: manifest ghost rune uses 40% of your blood

**Comment by kilozombie** [comment]
It's fundamentally flawed that you can make minions that have 200 health, and are unstunnable, at the cost of damage that can be healed easily.
Bicaridine IV, diona, multiple cultists, whatever. If there's no actual consequence for a manifest goat dying, then it will always be unrewarding to kill them, seeing as the real cultist has succeeded with minor injuries.

edit: You can even blood drain manifest ghosts. You can singlehandedly summon Nar-sie with just manifest ghosts and blood drains.

**Comment by AutismSpider** [comment]
if you do take any real damage you can use ghost or two to blood drain and heal all the damage back up.

**Comment by Blithering** [comment]
Alternatively, making manifested ghosts into a construct type could help against it. People would be less inclined to attempt stunning and they'd be unable to use complex equipment like coffee machines.

**Comment by eplgr** [comment]
Cooldown is an awful idea, I mean, cult already has many problems and they definetely don't need a random nerf. Ghosts/other players moving you out of the rune and killing all other manifested ghosts happens very often. 

But I do think overall it's really imbalanced. I've seen someone summoning 8 ghosts and nar'sie all by himself (drain runes helped). Yeah, as a **human**. 

I think the nerfs need to be put on hold until we/someone rebalances how cult works overall.

**Comment by despotate** [comment]
the ghost dies, you get critted

**Comment by SonixApache** [comment]
dionae can regen out of crit
ghost dies; you die IRL

The proper solution would be to just remove manifested ghosts.

**Comment by Allspice** [comment]
Remove the ghosts and buff shit like Raise Dead and Tele Other.

**Comment by murnandreams** [comment]
Hi! So yeah, I suggest just making it burn damage. It will fuck up diona who use it, thereby eliminating that powergame strat.

**Comment by murnandreams** [comment]
Figured I'd drive the whole manifest ghost shit into the ground, until it gets fixed. Pretty good way to make things happen.

**Comment by uazuaz** [comment]
when you summon a manifested ghost, there could be a cooldown before you can summon another ghost; you can still summon multiple ghosts at once.

drain rune could not be used anymore to succ manifested ghosts.

burn damage to dionae when they summon manifested ghosts.

also, dionae don't have blood; if they use sap to draw runes it should take much more time.

**Comment by Shadowmech88** [comment]
What if only one ghost could be summoned per rune, and the ghost was bound within a certain range of that rune, unable to travel outside it?
Erasing the rune would instantly kill the manifested ghost.

**Comment by kilozombie** [comment]
Still easily gamable.
Hide in a room in Maint, get 8 runes, summon 8 ghosts, use two of them to blood drain and keep making more (if you're not diona) and send the other 6 out with armor and weapons.

**Comment by Shadowmech88** [comment]
Yeah but at least it'd turn into a fortress defense instead of a send-invincible-murderghosts-to-ransack-station since the ghosts would have a limit on their range.

**Comment by kilozombie** [comment]
Definitely.

**Comment by abagone** [comment]
What if the rune costed a lot of blood like maybe 150-300 units to summon not sustain the manifest ghost.

And the manifest ghost needed to constantly drink blood to survive and have the ability to bite people like vampires or drink out of cups. 

One other thing they should take a lot of burn damage like 70-90 on contact from the bible or holy water and 5-15 on hallowed ground. 
And Null rod and space disintegrates them.

Basically my idea boils down to turning them mini vampires with no powers.

**Comment by Allspice** [comment]
Would all of that be fun though

**Comment by Intigracy** [comment]
Make the user of the rune not cost blood to use the rune.
Except if you hurt the ghost it actually hurts the rune user.
And you can only summon one ghost at a time per person.

**Comment by kilozombie** [comment]
>Except if you hurt the ghost it actually hurts the rune user.

Diona with a keloderm IV

**Comment by Intigracy** [comment]
doesn't outheal a laser gun being emptied into them

**Comment by AutismSpider** [comment]
like four laser gun hits to kill a diona
